services/user.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib.auth.hashers import make_password
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> User:
+    user = User.objects.create_user(username=username, password=password)

```

Comment:
The `create_user` method is correctly used to create a new user with a hashed password, which is a good practice for security.